DNN Platform/DotNetNuke.Web.Mvc/Startup.cs

```diff
@@ -13,11 +13,12 @@ public class Startup : IDnnStartup
     {
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddSingleton(serviceProvider => ControllerBuilder.Current.GetControllerFactory());
+            services.AddSingleton<IControllerFactory, DefaultControllerFactory>();
```

Comment:
Can you explain the intent of this change?

Reply:
This is explained in the linked issue. It triggers a recursive call inside the `ControllerBuilder` and throws an error.

Reply:
This makes sense, thanks for the explanation